skimage/transform/tests/test_geometric.py

```diff
@@ -49,10 +49,9 @@ def test_estimate_transform():
 
 
 def test_matrix_transform():
-    tform = AffineTransform(scale=(0.1, 0.5), rotation=2)
+    tform = AffineTransform(scale=(0.1, 0.5), rotation=2, shear=(0.5, 2))
```

Comment:
Test both versions of shear, and make sure the pixels end up where they are supposed to.  This can be done by, e.g., having a white pixel in an image, warping it, and making sure that white pixel ends up in the right place (use interpolation oder = 0)